### ISSUE TITLE
Transpiling zstd fails while compiling zstd succeeds

### ISSUE THREAD

**Issue by NobodyXu** [immunant/c2rust#1193]
Steps to reproduce:

Create a container using rust:slim from docker registry (sha256:393571e03852d16c3d71f5b55d9d8c55e90a08f65ce94872fb58a5626a98df0f), then install c2rust:

```bash
apt install build-essential llvm clang libclang-dev cmake libssl-dev pkg-config python3 git
cargo install c2rust
```

```bash
git clone https://github.com/facebook/zstd
git checkout v1.5.6

cd zstd/build/cmake

cmake -DCMAKE_EXPORT_COMPILE_COMMANDS=1 \
  -DZSTD_MULTITHREAD_SUPPORT=1 \
  -DZSTD_LEGACY_SUPPORT=1 \
  .
c2rust transpile -e compile_commands.json
```

<details>

<summary>
Output:
</summary>

```
[33mwarning:[0m Skipping duplicate compilation cmd for /zstd-rs/zstd-safe/zstd-sys/zstd/lib/common/debug.c
[33mwarning:[0m Skipping duplicate compilation cmd for /zstd-rs/zstd-safe/zstd-sys/zstd/lib/common/entropy_common.c
[33mwarning:[0m Skipping duplicate compilation cmd for /zstd-rs/zstd-safe/zstd-sys/zstd/lib/common/error_private.c
[33mwarning:[0m Skipping duplicate compilation cmd for /zstd-rs/zstd-safe/zstd-sys/zstd/lib/common/fse_decompress.c
[33mwarning:[0m Skipping duplicate compilation cmd for /zstd-rs/zstd-safe/zstd-sys/zstd/lib/common/pool.c
[33mwarning:[0m Skipping duplicate compilation cmd for /zstd-rs/zstd-safe/zstd-sys/zstd/lib/common/threading.c
[33mwarning:[0m Skipping duplicate compilation cmd for /zstd-rs/zstd-safe/zstd-sys/zstd/lib/common/xxhash.c
[33mwarning:[0m Skipping duplicate compilation cmd for /zstd-rs/zstd-safe/zstd-sys/zstd/lib/common/zstd_common.c
[33mwarning:[0m Skipping duplicate compilation cmd for /zstd-rs/zstd-safe/zstd-sys/zstd/lib/compress/fse_compress.c
[33mwarning:[0m Skipping duplicate compilation cmd for /zstd-rs/zstd-safe/zstd-sys/zstd/lib/compress/hist.c
[33mwarning:[0m Skipping duplicate compilation cmd for /zstd-rs/zstd-safe/zstd-sys/zstd/lib/compress/huf_compress.c
[33mwarning:[0m Skipping duplicate compilation cmd for /zstd-rs/zstd-safe/zstd-sys/zstd/lib/compress/zstd_compress.c
[33mwarning:[0m Skipping duplicate compilation cmd for /zstd-rs/zstd-safe/zstd-sys/zstd/lib/compress/zstd_compress_literals.c
[33mwarning:[0m Skipping duplicate compilation cmd for /zstd-rs/zstd-safe/zstd-sys/zstd/lib/compress/zstd_compress_sequences.c
[33mwarning:[0m Skipping duplicate compilation cmd for /zstd-rs/zstd-safe/zstd-sys/zstd/lib/compress/zstd_compress_superblock.c
[33mwarning:[0m Skipping duplicate compilation cmd for /zstd-rs/zstd-safe/zstd-sys/zstd/lib/compress/zstd_double_fast.c
[33mwarning:[0m Skipping duplicate compilation cmd for /zstd-rs/zstd-safe/zstd-sys/zstd/lib/compress/zstd_fast.c
[33mwarning:[0m Skipping duplicate compilation cmd for /zstd-rs/zstd-safe/zstd-sys/zstd/lib/compress/zstd_lazy.c
[33mwarning:[0m Skipping duplicate compilation cmd for /zstd-rs/zstd-safe/zstd-sys/zstd/lib/compress/zstd_ldm.c
[33mwarning:[0m Skipping duplicate compilation cmd for /zstd-rs/zstd-safe/zstd-sys/zstd/lib/compress/zstd_opt.c
[33mwarning:[0m Skipping duplicate compilation cmd for /zstd-rs/zstd-safe/zstd-sys/zstd/lib/compress/zstdmt_compress.c
[33mwarning:[0m Skipping duplicate compilation cmd for /zstd-rs/zstd-safe/zstd-sys/zstd/lib/decompress/huf_decompress.c
[33mwarning:[0m Skipping duplicate compilation cmd for /zstd-rs/zstd-safe/zstd-sys/zstd/lib/decompress/zstd_ddict.c
[33mwarning:[0m Skipping duplicate compilation cmd for /zstd-rs/zstd-safe/zstd-sys/zstd/lib/decompress/zstd_decompress.c
[33mwarning:[0m Skipping duplicate compilation cmd for /zstd-rs/zstd-safe/zstd-sys/zstd/lib/decompress/zstd_decompress_block.c
[33mwarning:[0m Skipping duplicate compilation cmd for /zstd-rs/zstd-safe/zstd-sys/zstd/lib/dictBuilder/cover.c
[33mwarning:[0m Skipping duplicate compilation cmd for /zstd-rs/zstd-safe/zstd-sys/zstd/lib/dictBuilder/divsufsort.c
[33mwarning:[0m Skipping duplicate compilation cmd for /zstd-rs/zstd-safe/zstd-sys/zstd/lib/dictBuilder/fastcover.c
[33mwarning:[0m Skipping duplicate compilation cmd for /zstd-rs/zstd-safe/zstd-sys/zstd/lib/dictBuilder/zdict.c
[33mwarning:[0m Skipping duplicate compilation cmd for /zstd-rs/zstd-safe/zstd-sys/zstd/lib/legacy/zstd_v01.c
[33mwarning:[0m Skipping duplicate compilation cmd for /zstd-rs/zstd-safe/zstd-sys/zstd/lib/legacy/zstd_v02.c
[33mwarning:[0m Skipping duplicate compilation cmd for /zstd-rs/zstd-safe/zstd-sys/zstd/lib/legacy/zstd_v03.c
[33mwarning:[0m Skipping duplicate compilation cmd for /zstd-rs/zstd-safe/zstd-sys/zstd/lib/legacy/zstd_v04.c
[33mwarning:[0m Skipping duplicate compilation cmd for /zstd-rs/zstd-safe/zstd-sys/zstd/lib/legacy/zstd_v05.c
[33mwarning:[0m Skipping duplicate compilation cmd for /zstd-rs/zstd-safe/zstd-sys/zstd/lib/legacy/zstd_v06.c
[33mwarning:[0m Skipping duplicate compilation cmd for /zstd-rs/zstd-safe/zstd-sys/zstd/lib/legacy/zstd_v07.c
[33mwarning:[0m Skipping duplicate compilation cmd for /zstd-rs/zstd-safe/zstd-sys/zstd/programs/zstdcli.c
[33mwarning:[0m Skipping duplicate compilation cmd for /zstd-rs/zstd-safe/zstd-sys/zstd/programs/util.c
[33mwarning:[0m Skipping duplicate compilation cmd for /zstd-rs/zstd-safe/zstd-sys/zstd/programs/timefn.c
[33mwarning:[0m Skipping duplicate compilation cmd for /zstd-rs/zstd-safe/zstd-sys/zstd/programs/fileio.c
[33mwarning:[0m Skipping duplicate compilation cmd for /zstd-rs/zstd-safe/zstd-sys/zstd/programs/fileio_asyncio.c
Transpiling debug.c
Transpiling entropy_common.c
Transpiling error_private.c
Transpiling fse_decompress.c
Transpiling pool.c
Transpiling threading.c
Transpiling xxhash.c
error: gnoring static assert during translation
error: Failed to translate ZSTD_XXH32_update: Unimplemented builtin __builtin_assume
--> /zstd-rs/zstd-safe/zstd-sys/zstd/lib/common/xxhash.h:3082:9

error: Failed to translate ZSTD_XXH32_reset: Unimplemented builtin __builtin_assume
--> /zstd-rs/zstd-safe/zstd-sys/zstd/lib/common/xxhash.h:3067:5

error: Failed to translate XXH32_finalize: Unimplemented builtin __builtin_assume
--> /zstd-rs/zstd-safe/zstd-sys/zstd/lib/common/xxhash.h:2915:20

error: Failed to translate XXH32_endian_align: Unimplemented builtin __builtin_assume
--> /zstd-rs/zstd-safe/zstd-sys/zstd/lib/common/xxhash.h:2995:22

error: Failed to translate XXH64_finalize: Unimplemented builtin __builtin_assume
--> /zstd-rs/zstd-safe/zstd-sys/zstd/lib/common/xxhash.h:3392:20

error: Failed to translate XXH64_endian_align: Unimplemented builtin __builtin_assume
--> /zstd-rs/zstd-safe/zstd-sys/zstd/lib/common/xxhash.h:3437:22

error: Failed to translate ZSTD_XXH64_reset: Unimplemented builtin __builtin_assume
--> /zstd-rs/zstd-safe/zstd-sys/zstd/lib/common/xxhash.h:3513:5

error: Failed to translate ZSTD_XXH64_update: Unimplemented builtin __builtin_assume
--> /zstd-rs/zstd-safe/zstd-sys/zstd/lib/common/xxhash.h:3527:9

warning: ignoring static assert during translation
Transpiling zstd_common.c
Transpiling fse_compress.c
Transpiling hist.c
Transpiling huf_compress.c
Transpiling zstd_compress.c
error: Skipping declaration Some(Typedef { name: "int8x8_t", typ: CQualTypeId { qualifiers: Qualifiers { is_const: false, is_restrict: false, is_volatile: false }, ctype: CTypeId(1603) }, is_implicit: false }) due to error: Unsupported type Vector(CQualTypeId { qualifiers: Qualifiers { is_const: false, is_restrict: false, is_volatile: false }, ctype: CTypeId(1604) }, 8)

error: Skipping declaration Some(Typedef { name: "int8x16_t", typ: CQualTypeId { qualifiers: Qualifiers { is_const: false, is_restrict: false, is_volatile: false }, ctype: CTypeId(1606) }, is_implicit: false }) due to error: Unsupported type Vector(CQualTypeId { qualifiers: Qualifiers { is_const: false, is_restrict: false, is_volatile: false }, ctype: CTypeId(1604) }, 16)

error: Skipping declaration Some(Typedef { name: "uint8x8_t", typ: CQualTypeId { qualifiers: Qualifiers { is_const: false, is_restrict: false, is_volatile: false }, ctype: CTypeId(1622) }, is_implicit: false }) due to error: Unsupported type Vector(CQualTypeId { qualifiers: Qualifiers { is_const: false, is_restrict: false, is_volatile: false }, ctype: CTypeId(1595) }, 8)

error: Skipping declaration Some(Typedef { name: "uint8x16_t", typ: CQualTypeId { qualifiers: Qualifiers { is_const: false, is_restrict: false, is_volatile: false }, ctype: CTypeId(1624) }, is_implicit: false }) due to error: Unsupported type Vector(CQualTypeId { qualifiers: Qualifiers { is_const: false, is_restrict: false, is_volatile: false }, ctype: CTypeId(1595) }, 16)

error: Failed to translate ZSTD_copy8: Unsupported vector default initializer: Typedef(CDeclId(1557)) x 8

error: Failed to translate ZSTD_copy16: Unsupported vector default initializer: Typedef(CDeclId(1557)) x 16

Transpiling zstd_compress_literals.c
Transpiling zstd_compress_sequences.c
Transpiling zstd_compress_superblock.c
Transpiling zstd_double_fast.c
error: Skipping declaration Some(Typedef { name: "int8x8_t", typ: CQualTypeId { qualifiers: Qualifiers { is_const: false, is_restrict: false, is_volatile: false }, ctype: CTypeId(276) }, is_implicit: false }) due to error: Unsupported type Vector(CQualTypeId { qualifiers: Qualifiers { is_const: false, is_restrict: false, is_volatile: false }, ctype: CTypeId(277) }, 8)

error: Skipping declaration Some(Typedef { name: "int8x16_t", typ: CQualTypeId { qualifiers: Qualifiers { is_const: false, is_restrict: false, is_volatile: false }, ctype: CTypeId(279) }, is_implicit: false }) due to error: Unsupported type Vector(CQualTypeId { qualifiers: Qualifiers { is_const: false, is_restrict: false, is_volatile: false }, ctype: CTypeId(277) }, 16)

error: Skipping declaration Some(Typedef { name: "uint8x8_t", typ: CQualTypeId { qualifiers: Qualifiers { is_const: false, is_restrict: false, is_volatile: false }, ctype: CTypeId(296) }, is_implicit: false }) due to error: Unsupported type Vector(CQualTypeId { qualifiers: Qualifiers { is_const: false, is_restrict: false, is_volatile: false }, ctype: CTypeId(268) }, 8)

error: Skipping declaration Some(Typedef { name: "uint8x16_t", typ: CQualTypeId { qualifiers: Qualifiers { is_const: false, is_restrict: false, is_volatile: false }, ctype: CTypeId(298) }, is_implicit: false }) due to error: Unsupported type Vector(CQualTypeId { qualifiers: Qualifiers { is_const: false, is_restrict: false, is_volatile: false }, ctype: CTypeId(268) }, 16)

error: Failed to translate ZSTD_copy8: Unsupported vector default initializer: Typedef(CDeclId(230)) x 8

error: Failed to translate ZSTD_copy16: Unsupported vector default initializer: Typedef(CDeclId(230)) x 16

Transpiling zstd_fast.c
error: Skipping declaration Some(Typedef { name: "int8x8_t", typ: CQualTypeId { qualifiers: Qualifiers { is_const: false, is_restrict: false, is_volatile: false }, ctype: CTypeId(276) }, is_implicit: false }) due to error: Unsupported type Vector(CQualTypeId { qualifiers: Qualifiers { is_const: false, is_restrict: false, is_volatile: false }, ctype: CTypeId(277) }, 8)

error: Skipping declaration Some(Typedef { name: "int8x16_t", typ: CQualTypeId { qualifiers: Qualifiers { is_const: false, is_restrict: false, is_volatile: false }, ctype: CTypeId(279) }, is_implicit: false }) due to error: Unsupported type Vector(CQualTypeId { qualifiers: Qualifiers { is_const: false, is_restrict: false, is_volatile: false }, ctype: CTypeId(277) }, 16)

error: Skipping declaration Some(Typedef { name: "uint8x8_t", typ: CQualTypeId { qualifiers: Qualifiers { is_const: false, is_restrict: false, is_volatile: false }, ctype: CTypeId(296) }, is_implicit: false }) due to error: Unsupported type Vector(CQualTypeId { qualifiers: Qualifiers { is_const: false, is_restrict: false, is_volatile: false }, ctype: CTypeId(268) }, 8)

error: Skipping declaration Some(Typedef { name: "uint8x16_t", typ: CQualTypeId { qualifiers: Qualifiers { is_const: false, is_restrict: false, is_volatile: false }, ctype: CTypeId(298) }, is_implicit: false }) due to error: Unsupported type Vector(CQualTypeId { qualifiers: Qualifiers { is_const: false, is_restrict: false, is_volatile: false }, ctype: CTypeId(268) }, 16)

error: Failed to translate ZSTD_copy8: Unsupported vector default initializer: Typedef(CDeclId(230)) x 8

error: Failed to translate ZSTD_copy16: Unsupported vector default initializer: Typedef(CDeclId(230)) x 16

Transpiling zstd_lazy.c
error: Skipping declaration Some(Typedef { name: "int8x8_t", typ: CQualTypeId { qualifiers: Qualifiers { is_const: false, is_restrict: false, is_volatile: false }, ctype: CTypeId(276) }, is_implicit: false }) due to error: Unsupported type Vector(CQualTypeId { qualifiers: Qualifiers { is_const: false, is_restrict: false, is_volatile: false }, ctype: CTypeId(277) }, 8)

error: Skipping declaration Some(Typedef { name: "int8x16_t", typ: CQualTypeId { qualifiers: Qualifiers { is_const: false, is_restrict: false, is_volatile: false }, ctype: CTypeId(279) }, is_implicit: false }) due to error: Unsupported type Vector(CQualTypeId { qualifiers: Qualifiers { is_const: false, is_restrict: false, is_volatile: false }, ctype: CTypeId(277) }, 16)

error: Skipping declaration Some(Typedef { name: "int64x1_t", typ: CQualTypeId { qualifiers: Qualifiers { is_const: false, is_restrict: false, is_volatile: false }, ctype: CTypeId(291) }, is_implicit: false }) due to error: Unsupported type Vector(CQualTypeId { qualifiers: Qualifiers { is_const: false, is_restrict: false, is_volatile: false }, ctype: CTypeId(292) }, 1)

error: Skipping declaration Some(Typedef { name: "uint8x8_t", typ: CQualTypeId { qualifiers: Qualifiers { is_const: false, is_restrict: false, is_volatile: false }, ctype: CTypeId(296) }, is_implicit: false }) due to error: Unsupported type Vector(CQualTypeId { qualifiers: Qualifiers { is_const: false, is_restrict: false, is_volatile: false }, ctype: CTypeId(268) }, 8)

error: Skipping declaration Some(Typedef { name: "uint8x16_t", typ: CQualTypeId { qualifiers: Qualifiers { is_const: false, is_restrict: false, is_volatile: false }, ctype: CTypeId(298) }, is_implicit: false }) due to error: Unsupported type Vector(CQualTypeId { qualifiers: Qualifiers { is_const: false, is_restrict: false, is_volatile: false }, ctype: CTypeId(268) }, 16)

error: Skipping declaration Some(Typedef { name: "uint16x8_t", typ: CQualTypeId { qualifiers: Qualifiers { is_const: false, is_restrict: false, is_volatile: false }, ctype: CTypeId(302) }, is_implicit: false }) due to error: Unsupported type Vector(CQualTypeId { qualifiers: Qualifiers { is_const: false, is_restrict: false, is_volatile: false }, ctype: CTypeId(270) }, 8)

error: Skipping declaration Some(Typedef { name: "uint64x1_t", typ: CQualTypeId { qualifiers: Qualifiers { is_const: false, is_restrict: false, is_volatile: false }, ctype: CTypeId(309) }, is_implicit: false }) due to error: Unsupported type Vector(CQualTypeId { qualifiers: Qualifiers { is_const: false, is_restrict: false, is_volatile: false }, ctype: CTypeId(272) }, 1)

error: Failed to translate vceqq_u8: Unsupported vector default initializer: Typedef(CDeclId(230)) x 16

error: Failed to translate vdupq_n_u8: Unsupported vector default initializer: Typedef(CDeclId(230)) x 16

error: Failed to translate vreinterpretq_u8_u16: Unsupported vector default initializer: Typedef(CDeclId(230)) x 16

error: Failed to translate vreinterpretq_u16_u8: Unsupported vector default initializer: Typedef(CDeclId(231)) x 8

error: Failed to translate vreinterpret_u64_u8: Unsupported vector default initializer: Typedef(CDeclId(233)) x 1

error: Failed to translate ZSTD_copy8: Unsupported vector default initializer: Typedef(CDeclId(230)) x 8

error: Failed to translate ZSTD_copy16: Unsupported vector default initializer: Typedef(CDeclId(230)) x 16

error: Failed to translate ZSTD_row_getNEONMask: Unsupported vector default initializer: Typedef(CDeclId(230)) x 16

Transpiling zstd_ldm.c
error: Skipping declaration Some(Typedef { name: "int8x8_t", typ: CQualTypeId { qualifiers: Qualifiers { is_const: false, is_restrict: false, is_volatile: false }, ctype: CTypeId(276) }, is_implicit: false }) due to error: Unsupported type Vector(CQualTypeId { qualifiers: Qualifiers { is_const: false, is_restrict: false, is_volatile: false }, ctype: CTypeId(277) }, 8)

error: Skipping declaration Some(Typedef { name: "int8x16_t", typ: CQualTypeId { qualifiers: Qualifiers { is_const: false, is_restrict: false, is_volatile: false }, ctype: CTypeId(279) }, is_implicit: false }) due to error: Unsupported type Vector(CQualTypeId { qualifiers: Qualifiers { is_const: false, is_restrict: false, is_volatile: false }, ctype: CTypeId(277) }, 16)

error: Skipping declaration Some(Typedef { name: "uint8x8_t", typ: CQualTypeId { qualifiers: Qualifiers { is_const: false, is_restrict: false, is_volatile: false }, ctype: CTypeId(296) }, is_implicit: false }) due to error: Unsupported type Vector(CQualTypeId { qualifiers: Qualifiers { is_const: false, is_restrict: false, is_volatile: false }, ctype: CTypeId(268) }, 8)

error: Skipping declaration Some(Typedef { name: "uint8x16_t", typ: CQualTypeId { qualifiers: Qualifiers { is_const: false, is_restrict: false, is_volatile: false }, ctype: CTypeId(298) }, is_implicit: false }) due to error: Unsupported type Vector(CQualTypeId { qualifiers: Qualifiers { is_const: false, is_restrict: false, is_volatile: false }, ctype: CTypeId(268) }, 16)

error: Failed to translate ZSTD_copy8: Unsupported vector default initializer: Typedef(CDeclId(230)) x 8

error: Failed to translate ZSTD_copy16: Unsupported vector default initializer: Typedef(CDeclId(230)) x 16

Transpiling zstd_opt.c
error: Skipping declaration Some(Typedef { name: "int8x8_t", typ: CQualTypeId { qualifiers: Qualifiers { is_const: false, is_restrict: false, is_volatile: false }, ctype: CTypeId(276) }, is_implicit: false }) due to error: Unsupported type Vector(CQualTypeId { qualifiers: Qualifiers { is_const: false, is_restrict: false, is_volatile: false }, ctype: CTypeId(277) }, 8)

error: Skipping declaration Some(Typedef { name: "int8x16_t", typ: CQualTypeId { qualifiers: Qualifiers { is_const: false, is_restrict: false, is_volatile: false }, ctype: CTypeId(279) }, is_implicit: false }) due to error: Unsupported type Vector(CQualTypeId { qualifiers: Qualifiers { is_const: false, is_restrict: false, is_volatile: false }, ctype: CTypeId(277) }, 16)

error: Skipping declaration Some(Typedef { name: "uint8x8_t", typ: CQualTypeId { qualifiers: Qualifiers { is_const: false, is_restrict: false, is_volatile: false }, ctype: CTypeId(296) }, is_implicit: false }) due to error: Unsupported type Vector(CQualTypeId { qualifiers: Qualifiers { is_const: false, is_restrict: false, is_volatile: false }, ctype: CTypeId(268) }, 8)

error: Skipping declaration Some(Typedef { name: "uint8x16_t", typ: CQualTypeId { qualifiers: Qualifiers { is_const: false, is_restrict: false, is_volatile: false }, ctype: CTypeId(298) }, is_implicit: false }) due to error: Unsupported type Vector(CQualTypeId { qualifiers: Qualifiers { is_const: false, is_restrict: false, is_volatile: false }, ctype: CTypeId(268) }, 16)

error: Failed to translate ZSTD_copy8: Unsupported vector default initializer: Typedef(CDeclId(230)) x 8

error: Failed to translate ZSTD_copy16: Unsupported vector default initializer: Typedef(CDeclId(230)) x 16

Transpiling zstdmt_compress.c
Transpiling huf_decompress.c
Transpiling zstd_ddict.c
Transpiling zstd_decompress.c
Transpiling zstd_decompress_block.c
error: Skipping declaration Some(Typedef { name: "int8x8_t", typ: CQualTypeId { qualifiers: Qualifiers { is_const: false, is_restrict: false, is_volatile: false }, ctype: CTypeId(540) }, is_implicit: false }) due to error: Unsupported type Vector(CQualTypeId { qualifiers: Qualifiers { is_const: false, is_restrict: false, is_volatile: false }, ctype: CTypeId(541) }, 8)

error: Skipping declaration Some(Typedef { name: "int8x16_t", typ: CQualTypeId { qualifiers: Qualifiers { is_const: false, is_restrict: false, is_volatile: false }, ctype: CTypeId(543) }, is_implicit: false }) due to error: Unsupported type Vector(CQualTypeId { qualifiers: Qualifiers { is_const: false, is_restrict: false, is_volatile: false }, ctype: CTypeId(541) }, 16)

error: Skipping declaration Some(Typedef { name: "uint8x8_t", typ: CQualTypeId { qualifiers: Qualifiers { is_const: false, is_restrict: false, is_volatile: false }, ctype: CTypeId(560) }, is_implicit: false }) due to error: Unsupported type Vector(CQualTypeId { qualifiers: Qualifiers { is_const: false, is_restrict: false, is_volatile: false }, ctype: CTypeId(532) }, 8)

error: Skipping declaration Some(Typedef { name: "uint8x16_t", typ: CQualTypeId { qualifiers: Qualifiers { is_const: false, is_restrict: false, is_volatile: false }, ctype: CTypeId(562) }, is_implicit: false }) due to error: Unsupported type Vector(CQualTypeId { qualifiers: Qualifiers { is_const: false, is_restrict: false, is_volatile: false }, ctype: CTypeId(532) }, 16)

error: Failed to translate ZSTD_copy16: Unsupported vector default initializer: Typedef(CDeclId(494)) x 16

error: Failed to translate ZSTD_copy8: Unsupported vector default initializer: Typedef(CDeclId(494)) x 8

Transpiling cover.c
Transpiling divsufsort.c
Transpiling fastcover.c
Transpiling zdict.c
Transpiling zstd_v01.c
Transpiling zstd_v02.c
Transpiling zstd_v03.c
Transpiling zstd_v04.c
Transpiling zstd_v05.c
Transpiling zstd_v06.c
Transpiling zstd_v07.c
Transpiling benchfn.c
Transpiling benchzstd.c
rrror: Failed to translate XXH64_finalize: Unimplemented builtin __builtin_assume
--> /zstd-rs/zstd-safe/zstd-sys/zstd/lib/common/xxhash.h:3392:20

error: Failed to translate XXH64_endian_align: Unimplemented builtin __builtin_assume
--> /zstd-rs/zstd-safe/zstd-sys/zstd/lib/common/xxhash.h:3437:22

Transpiling datagen.c
Transpiling dibio.c
Transpiling fileio.c
Transpiling fileio_asyncio.c
Transpiling lorem.c
Transpiling timefn.c
Transpiling util.c
Transpiling zstdcli.c
Transpiling zstdcli_trace.c
```
</details>

TL;DR:
 - Unimplemented builtin `__builtin_assume`
 - Unsupported vector default initializer: `Typedef(CDeclId(494)) x 16`

**Comment by NobodyXu** [comment]
According to Trevor Gross, `__builtin_assume` can be translated to `hint::assert_unchecked`
